database/pom.xml

```diff
@@ -17,7 +17,7 @@
   <properties>
 
     <!-- database driver versions -->
-    <version.h2>2.1.214</version.h2>
+    <version.h2>2.2.222</version.h2>
```

Comment:
❌ When we bump H2 anyways, let's bump to the latest version.

Reply:
Updated to latest `2.3.232` (currently)